From 9a6098e55d9b7f00f087412fac89d31534579d67 Mon Sep 17 00:00:00 2001
From: arakiken <arakiken@users.sf.net>
Date: Sun, 7 Sep 2025 02:52:42 +0900
Subject: [PATCH] * README.cocoa, README.cocoatouch: Updated. * ui_shortcut.c: 
  Change the default values of the following shortcuts on MacOSX.   -
 INSERT_CLIPBOARD = Command+v   - COPY_CLIPBOARD = Command+c  
 (https://github.com/arakiken/mlterm/issues/150)

diff --git a/uitoolkit/ui_shortcut.c b/uitoolkit/ui_shortcut.c
index 553430d4..409738c2 100644
--- uitoolkit/ui_shortcut.c
+++ uitoolkit/ui_shortcut.c
@@ -176,9 +176,18 @@ void ui_shortcut_init(ui_shortcut_t *shortcut) {
 #endif
 
     /* INSERT_CLIPBOARD */
+#if defined(USE_QUARTZ) || (defined(USE_SDL2) && defined(__APPLE__))
+    { 'v', CommandMask, 1, },
+#else
     { 'v', ControlMask|ShiftMask, 1, },
+#endif
+
     /* COPY_CLIPBOARD */
+#if defined(USE_QUARTZ) || (defined(USE_SDL2) && defined(__APPLE__))
+    { 'c', CommandMask, 1, },
+#else
     { 'c', ControlMask|ShiftMask, 1, },
+#endif
 
     /* RESET */
     { XK_Pause, 0, 1, },
